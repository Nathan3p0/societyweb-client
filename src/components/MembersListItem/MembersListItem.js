import React from 'react';

const MembersListItem = (props) => {
    return (
        <li className="tableRow">
            <div className="col col-1" data-label="Full Name">{props.fullname}</div>
            <div className="col col-2" data-label="Username">{props.username}</div>
            <div className="col col-3" data-label="Phone Number">{props.phone}</div>
            <div className="col col-4" data-label="Email">{props.email}</div>
        </li>
    );
}

export default MembersListItem;