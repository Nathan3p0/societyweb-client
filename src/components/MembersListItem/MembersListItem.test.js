import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import MembersListItem from './MembersListItem'

describe('Members List Item Component Rendering Tests', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<MembersListItem />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it('renders to UI as expected', () => {
        const tree = renderer.create(<MembersListItem />).toJSON();
        expect(tree).toMatchSnapshot()
    })
})