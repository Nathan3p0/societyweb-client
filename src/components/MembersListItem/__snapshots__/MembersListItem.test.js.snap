// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Members List Item Component Rendering Tests renders to UI as expected 1`] = `
<li
  className="tableRow"
>
  <div
    className="col col-1"
    data-label="Full Name"
  />
  <div
    className="col col-2"
    data-label="Username"
  />
  <div
    className="col col-3"
    data-label="Phone Number"
  />
  <div
    className="col col-4"
    data-label="Email"
  />
</li>
`;
